python/ray/train/lightning/_lightning_utils.py

```diff
@@ -21,12 +21,21 @@
 LIGHTNING_REPORT_STAGE_KEY = "_report_on"
 
 
+def _get_worker_root_device():
+    """Get a single torch device for the current worker."""
```

Comment:
I guess you can just say that it defaults to the first GPU if there are multiple?

Reply:
Sounds good.